src/main/java/org/sagebionetworks/bridge/models/assessments/AssessmentDto.java

```diff
@@ -0,0 +1,160 @@
+package org.sagebionetworks.bridge.models.assessments;
+
+import static org.sagebionetworks.bridge.models.TagUtils.toStringSet;
+
+import java.util.Map;
+import java.util.Set;
+
+import org.joda.time.DateTime;
+
+import org.sagebionetworks.bridge.json.BridgeTypeName;
+
+/**
+ * The assessment as it is expressed in our REST API.
+ */
+@BridgeTypeName("Assessment")
+public class AssessmentDto {
+    
+    public static AssessmentDto create(Assessment assessment) {
+        AssessmentDto dto = new AssessmentDto();
+        dto.setGuid(assessment.getGuid());
+        dto.setIdentifier(assessment.getIdentifier());
+        dto.setRevision(assessment.getRevision());
+        dto.setTitle(assessment.getTitle());
+        dto.setSummary(assessment.getSummary());
+        dto.setValidationStatus(assessment.getValidationStatus());
+        dto.setNormingStatus(assessment.getNormingStatus());
+        dto.setOsName(assessment.getOsName());
+        dto.setOriginGuid(assessment.getOriginGuid());
+        dto.setOwnerId(assessment.getOwnerId());
+        dto.setTags(toStringSet(assessment.getTags()));    
+        dto.setCategories(toStringSet(assessment.getCategories()));
+        dto.setCustomizationFields(assessment.getCustomizationFields());
+        dto.setCreatedOn(assessment.getCreatedOn());
+        dto.setModifiedOn(assessment.getModifiedOn());
+        dto.setDeleted(assessment.isDeleted());
+        dto.setVersion(assessment.getVersion());
+        return dto;
+    }
+    
+    private String guid;
+    private String identifier;
+    private int revision = 1;
+    private String ownerId;
+    private String title;
+    private String summary;
+    private String osName;
+    private String originGuid;
+    private String validationStatus;
+    private String normingStatus;
```

Comment:
What do these status objects mean? Should they be enums? If these are intended to be machine readable, we probably don't want them to be freeform fields.

Reply:
These are indeed string values and they come out of the UI designs for assessments, where a description of validation and norming are part of the "list" views of assessments.

Reply:
Are these the same values that the data validation will use to determine validation mode? Or is that a separate thing to be added later?

Reply:
Yes these have nothing to do with validating the data, they are about whether or not the scientists have validated that the assessments measure what they purport to measure, for example, that the Samsung blood pressure measurement matches what you'd get at any given time from a blood pressure cuff. (That's validation, norming is similar but I'm not sure what it is.)